docs/src/main/docbook/declarative-linking.xml

```diff
@@ -262,6 +262,24 @@ List&lt;Link&gt; links</programlisting>
             Multiple link headers can be added by use of the <literal>@InjectLinks</literal> annotation
             which can contain multiple <literal>@InjectLink</literal> annotations.
         </para>
+      </section>
```

Comment:
Hello !

   I was going to update this only file but after checking, it appears that a lot (a lot !) of files still have the old copyright year all over the code base.

   Would you like me to update all of them in one commit or just the ones related to this PR?

Aurélien